imports/startup/server/accounts.js

```diff
@@ -51,9 +69,11 @@ if (Meteor.users.find().fetch().length === 0) {
     }
   ];
 
+  
+
   const Bio = "greatest 3d modeller on the planet";
   _.each(users, userData => {
-    const password = Meteor.settings.adminPassword;
+    const password = randomPassword;
```

Comment:
Add an if condition here.
If Meteor.settings.adminPassword equals "password"
Set password to randomPassword
Else
Set password to Meteor.settings.adminPassword

After if else statement, Console log the password variable here.